src/YieldDistributor.sol

```diff
@@ -49,6 +49,8 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
     mapping(address => uint256[]) voterDistributions;
     /// @notice How much of the yield is divided equally among projects
     uint256 public yieldFixedSplitDivisor;
+    /// @notice The address of the $BUTTEREDBREAD token contract
```

Comment:
`$BUTTEREDBREAD` implies it's the token symbol because of `$` character. Should change this to `The address of the \`ButteredBread\` token contract`

Reply:
addressed with 5fe2c80